SimPEG/EM/FDEM/ProblemFDEM.py

```diff
@@ -361,20 +441,41 @@ def getADeriv(self, freq, u, v, adjoint=False):
         MeSigmaIDeriv = MeSigmaIDeriv(vec)
 
         if adjoint:
-            if self._makeASymmetric is True:
-                v = MfMui * v
             return MeSigmaIDeriv.T * (C.T * v)
-
-        if self._makeASymmetric is True:
-            return MfMui.T * (C * (MeSigmaIDeriv * v))
         return C * (MeSigmaIDeriv * v)
 
+    def getADeriv_mui(self, freq, u, v, adjoint=False):
+
+        MfMui = self.MfMui
+        MfMuiDeriv = self.MfMuiDeriv(u)
+        MeSigmaI = self.MeSigmaI
+        C = self.mesh.edgeCurl
+
+        if adjoint:
+            return MfMuiDeriv.T * (C * (MeSigmaI.T * (C.T * v)))
+        return C * (MeSigmaI * (C.T * (MfMuiDeriv * v)))
+
+    def getADeriv(self, freq, u, v, adjoint=False):
+        if adjoint is True and self._makeASymmetric:
```

Comment:
I added a bunch of the checks/summary to PR #519 
Copied over the other comment to #464